apps/angular/55-back-button-navigation/project.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "angular-back-button-navigation",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "projectType": "application",
+  "prefix": "app",
+  "sourceRoot": "apps/angular/55-back-button-navigation/src",
+  "tags": [],
+  "targets": {
+    "build": {
+      "executor": "@angular-devkit/build-angular:application",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/apps/angular/55-back-button-navigation",
+        "index": "apps/angular/55-back-button-navigation/src/index.html",
+        "browser": "apps/angular/55-back-button-navigation/src/main.ts",
+        "polyfills": ["zone.js"],
+        "tsConfig": "apps/angular/55-back-button-navigation/tsconfig.app.json",
+        "inlineStyleLanguage": "scss",
+        "assets": [
+          {
+            "glob": "**/*",
+            "input": "apps/angular/55-back-button-navigation/public"
+          }
+        ],
+        "styles": ["apps/angular/55-back-button-navigation/src/styles.scss"],
```

Comment:
Angular core theme necessary or not?  Is there a warning in the browser with this setup?

`./node_modules/@angular/material/prebuilt-themes/indigo-pink.css`

Reply:
you could indeed add the default theme, but only the button and dialog theme are imported.

Reply:
yes, there is this warning with the setup provided.
<img width="729" alt="image" src="https://github.com/tomalaforge/angular-challenges/assets/7561447/366c4f9c-5336-4419-a464-fc3a6d55e2a8">
How to remove it? any suggestions?

Reply:
it has been a long time since I worked with Material, but you can import the default theme or try to find out what is missing. 😅

Reply:
I've just pushed a fix which removes the above warning. @jdegand please let me know, if this is what you proposed.

Reply:
Yes, that's why the core theme is necessary.